domains/tommi.json

```diff
@@ -0,0 +1,15 @@
+{
+  "owner": {
+    "username": "tommi",
+    "email": "surfing@tommi.space",
+    "GitHub": "xplosionmind",
+    "GitLab": "xplosionmind",
+    "Mastodon": "@tommi@pan.rent"
+  },
+	"repo": "https://codeberg.org/tommi/tommi.space",
+	"description": "The virtual representation of Tommi’s mind",
+  "record": {
+    "URL": "https://tommi.space",
+    "CNAME": "tommi.space"
```

Comment:
You cant use CNAME with any other  record (stick to either URL or CNAME)

Reply:
Sorry for the mistake. I am now using the URL record only.
Thanks!